src/HandlerInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\VarDumper;
+
+interface HandlerInterface
```

Comment:
```suggestion
interface FormatterInterface
```

Since this one doesn't handle anything but format the output. What do you think?